hw03_frequency_analysis/go.mod

```
@@ -1,4 +1,4 @@
-module github.com/fixme_my_friend/hw03_frequency_analysis
+module github.com/Deiklov/hw_otus/hw01_hello_otus
```

```suggestion
module github.com/Deiklov/hw_otus/hw03_frequency_analysis
```
У каждого задания свой собственный модуль